test/distributed/org/apache/cassandra/distributed/test/log/ClusterMetadataTestHelper.java

```diff
@@ -313,6 +318,18 @@ public static NodeId register(InetAddressAndPort endpoint, String dc, String rac
         }
     }
 
+    public static NodeId register(InetAddressAndPort endpoint, InetAddressAndPort endpointSSL, String dc, String rack)
```

Comment:
I think we could merge this together with register method above, the one above would just call this one with endpointSSL to be equal to endpoint